docs/api/en/textures/CompressedArrayTexture.html

```diff
@@ -49,10 +48,15 @@ <h2>Properties</h2>
 
 		<h3>[property:number wrapR]</h3>
 		<p>
-		This defines how the texture is wrapped in the depth direction.<br />
-		The default is [page:Textures THREE.ClampToEdgeWrapping], where the edge is clamped to the outer edge texels.
-		The other two choices are [page:Textures THREE.RepeatWrapping] and [page:Textures THREE.MirroredRepeatWrapping].
-		See the [page:Textures texture constants] page for details.
+			This defines how the texture is wrapped in the depth direction.<br />
+			The default is [page:Textures THREE.ClampToEdgeWrapping], where the edge is clamped to the outer edge texels.
+			The other two choices are [page:Textures THREE.RepeatWrapping] and [page:Textures THREE.MirroredRepeatWrapping].
+			See the [page:Textures texture constants] page for details.
+		</p>
+
+		<h3>[property:Image image]</h3>
+		<p>
+			Overridden with a record type containing width, height, and depth.
```

Comment:
The term "record type" sounds a bit unusual. Please use just "object".

Reply:
I agree.  It's not uncommon, but it's accurate.  Again, I was following what is currently in DataTexture and DataArrayTexture to make it match.  Update all 4 ?

Reply:
Yes please.